src/screens/LocationPage/utils.ts

```diff
@@ -22,7 +23,18 @@ export function getPageTitle(region: Region): string {
 
 export function getPageDescription(region: Region): string {
   const date = formatMetatagDate();
-  const { level: alarmLevel } = LocationSummariesByFIPS[region.fipsCode];
+  const { level: alarmLevel, metrics } = LocationSummariesByFIPS[
+    region.fipsCode
+  ];
   const levelInfo = LOCATION_SUMMARY_LEVELS[alarmLevel];
-  return `${date} COVID RISK LEVEL: ${levelInfo.detail(locationName(region))}`;
+  const vaccinationRatio = metrics[Metric.VACCINATIONS]?.value;
+  const vaccinationText = vaccinationRatio
+    ? ` and ${formatPercent(
+        vaccinationRatio,
+        1,
+      )} of the population has received at least one vaccine dose`
+    : '';
+  return `${date}: ${locationName(region)} is at ${
+    levelInfo.name
```

Comment:
maybe lowercase the level? "xxx is at Medium COVID risk level" --> "xxx is at medium COVID risk level"

Reply:
oops, good catch, thanks!